WORKSPACE.bazel

```diff
@@ -301,13 +301,8 @@ http_archive(
     name = "tla_apalache",
     build_file_content = """
 package(default_visibility = ["//visibility:public"])
-filegroup(
-  name = "contents",
-  srcs = [
-    "bin/apalache-mc",
-    "lib/apalache.jar",
-  ],
-)
+
+exports_files([ "bin/apalache-mc" ])
```

Comment:
I guess it works since it passes CI, but how does it work actually? The actual binary is the JAR, `apalache-mc` is just a wrapper script, how does the script find the JAR? I guess Bazel just turns the `apalache-mc` just into a symlink to the actual location, where the JAR is also available?

Reply:
yeah I did wonder about that as well and initially expected it to fail. Not 100% sure but I suspect that it just turns the `:bin/apalache-mc` target into a path to the original file (which is located next to the jar still)